csharp/private/net.BUILD

```diff
@@ -0,0 +1,501 @@
+load("//csharp/private:rules/nuget.bzl", "nuget_package")
+
+nuget_package(
```

Comment:
I think we'll need to plumb `workspace_file` from `nuget_package` down to `http_archive`. `nuget_package` only works in a `WORKSPACE` file.

But I like the idea of just embedding all this stuff inside the `@net` package!

Reply:
Yeah, actually, I really like this. Otherwise we might want to make of these packages only visible to `@net` explicitly which would be annoying, this hides them for free I think.